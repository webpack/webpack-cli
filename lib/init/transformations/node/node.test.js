"use strict";

const defineTest = require("../../../utils/defineTest");

defineTest(
	__dirname,
	"node",
	"node-0",
	{
		console: false,
		global: true,
		process: true,
		Buffer: true,
		__filename: "mock",
		__dirname: "mock",
		setImmediate: true
	},
	"init"
);
