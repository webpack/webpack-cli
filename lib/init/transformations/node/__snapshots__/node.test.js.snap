// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`node transforms correctly using "node-0" data 1`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    node: {
        console: false,
        global: true,
        process: true,
        Buffer: true,
        __filename: mock,
        __dirname: mock,
        setImmediate: true
    }
}
"
`;
